lightning/src/ln/channel.rs

```diff
@@ -12253,40 +12253,43 @@ mod tests {
 	}
 
 	#[test]
-	fn test_estimate_funding_transaction_fee() {
-		use crate::ln::channel::estimate_funding_transaction_fee;
+	fn test_estimate_fv2_unding_transaction_fee() {
+		use crate::ln::channel::estimate_v2_funding_transaction_fee;
 		use bitcoin::Weight;
 
-		let witness_weight = Weight::from_wu(300);
-
+		// 2 inputs with weight 300, initiator, 2000 sat/kw feerate
 		assert_eq!(
-			estimate_funding_transaction_fee(true, 2, witness_weight, 2000),
+			estimate_v2_funding_transaction_fee(true, 2, Weight::from_wu(300), 2000),
 			1668
 		);
 
+		// higher feerate
 		assert_eq!(
-			estimate_funding_transaction_fee(true, 2, witness_weight, 3000),
+			estimate_v2_funding_transaction_fee(true, 2, Weight::from_wu(300), 3000),
 			2502
 		);
 
+		// only 1 input
 		assert_eq!(
-			estimate_funding_transaction_fee(true, 1, witness_weight, 2000),
+			estimate_v2_funding_transaction_fee(true, 1, Weight::from_wu(300), 2000),
 			1348
 		);
 
+		// 0 input weight
 		assert_eq!(
-			estimate_funding_transaction_fee(true, 1, Weight::from_wu(0), 2000),
+			estimate_v2_funding_transaction_fee(true, 1, Weight::from_wu(0), 2000),
 			748
 		);
 
+		// not initiator
 		assert_eq!(
-			estimate_funding_transaction_fee(false, 1, Weight::from_wu(0), 2000),
+			estimate_v2_funding_transaction_fee(false, 1, Weight::from_wu(0), 2000),
 			320
 		);
 	}
 
-	fn prepare_input(value: u64) -> (TxIn, TransactionU16LenLimited) {
-		let input_1_prev_out = TxOut { value: Amount::from_sat(value), script_pubkey: ScriptBuf::default() };
+	fn prepare_funding_input(funding_input_sats: u64) -> (TxIn, TransactionU16LenLimited) {
```

Comment:
I'd much prefer `funding_input_sats`. When reading the test, "prepare" doesn't provide much useful information whereas "sats" helps us differentiate from msats.